content/40.inequality.md

```diff
@@ -73,31 +130,40 @@ In addition to Black Americans, disproportionate harm and mortality from COVID-1
 In Brazil, indigenous communities likewise carry an increased burden of COVID-19 [@doi:10.1126/science.abc0073].
 In the United Kingdom, nonwhite ethnicity (principally Black or South Asian) was one of several factors found to be associated with a higher risk of death from COVID-19 [@doi:10.1038/s41586-020-2521-4].
 
-From a genetic standpoint, it is highly unlikely that ancestry itself predisposes individuals to contracting COVID-19 or to experiencing severe COVID-19 outcomes.
-Examining human genetic diversity indicates variation over a geographic continuum, and that most human genetic variation is associated with the African continent [@doi:10.1038/ng1438].
+From a genetic standpoint, it is highly unlikely that ancestry itself predisposes individuals from these diverse groups to experiencing severe COVID-19 outcomes.
+Human genetic diversity varies over a geographic continuum, and that most human genetic variation is associated with the African continent [@doi:10.1038/ng1438].
 African-Americans are also a more genetically diverse group relative to European-Americans, with a large number of rare alleles and a much smaller fraction of common alleles identified in African-Americans [@doi:10.1146/annurev.genom.9.081307.164258].
-Therefore, the idea that African ancestry (at the continent level) might convey some sort of genetic risk for severe COVID-19 contrasts with what is known about worldwide human genetic diversity [@doi:10.1126/science.abd4842].
-The possibility for genetic variants that confer some risk or some protection remains possible, but has not been widely explored, especially at a global level.
+Therefore, the idea that African ancestry (at the continental level) might convey some sort of genetic risk for severe COVID-19 contrasts with what is known about worldwide human genetic diversity [@doi:10.1126/science.abd4842].
+The possibility for genetic variants that confer some risk or some protection remains possible, but has not been widely explored, especially on a global scale.<!--To Do: Check if there are updates here, I think I remember seeing some-->
 Research in Beijing of a small number (n=80) hospitalized COVID-19 patients revealed an association between severe COVID-19 outcomes and homozygosity for an allele in the interferon-induced transmembrane protein 3 (IFITM3) gene, which was selected as a candidate because it was previously found to be associated with influenza outcomes in Chinese patients [@doi:10.1093/infdis/jiaa224].
```

Comment:
```suggestion
Research in Beijing of a small number (n=80) of hospitalized COVID-19 patients revealed an association between severe COVID-19 outcomes and homozygosity for an allele in the interferon-induced transmembrane protein 3 (IFITM3) gene, which was selected as a candidate because it was previously found to be associated with influenza outcomes in Chinese patients [@doi:10.1093/infdis/jiaa224].
```
```suggestion
Research in Beijing of a small number (n=80) hospitalized COVID-19 patients revealed an association between severe COVID-19 outcomes and homozygosity for an allele in the interferon-induced transmembrane protein 3 (IFITM3) gene, which was selected as a candidate because it was previously found to be associated with influenza outcomes in Chinese patients [@doi:10.1093/infdis/jiaa224].
```